ci-operator/config/openshift/multiarch/openshift-multiarch-master__nightly-4.17-from-4.16.yaml

```diff
@@ -107,32 +94,32 @@ tests:
       OCP_ARCH: amd64
       TEST_SUITE: upgrade-conformance
     workflow: openshift-upgrade-azure-heterogeneous
-- as: ocp-e2e-upgrade-azure-ovn-arm64
+- as: ocp-e2e-upgrade-azure-ovn-multi-a-a
```

Comment:
can we drop 'ocp' from these names?

Reply:
Unfortunately, we can't.  I talked with @deepsm007 about this the other day, and unless I'm misremembering, there is automation expecting this.